test/parallel/test-preload.js

```diff
@@ -62,6 +63,23 @@ childProcess.exec(
   }
 );
 
+// Test that preload can be used with --frozen-intrinsics
+childProcess.exec(
+  `"${nodeBinary}" --frozen-intrinsics ${preloadOption([fixtureE])}-e "console.log('hello');"`,
+  function(err, stdout) {
+    assert.ifError(err);
+    assert.strictEqual(stdout, 'hello\n');
+  }
+);
+const workerSrc = `const {Worker} = require('worker_threads');new Worker(${JSON.stringify(fixtureA)});`;
```

Comment:
This doesn’t work (it needs `{ eval: true }` as an option) and won’t pass linting because of the line length

(It’s okay to use multiline template strings here)

Reply:
multiline string causes the shell string to break, i agree it looks better though

Reply:
reworked things a little to make stuff happy, still isn't the prettiest